apis/v0/jwt.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 Google LLC
+// Copyright 2023 Google LLC
```

Comment:
btw it's not required to keep the year up to date unless you're adding a new file

Reply:
Got it. Thank you!